File: favourite-lyrics-test/src/test/java/org/jesperancinha/lyrics/test/MainTest.java
package org.jesperancinha.lyrics.test;

import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;

import static org.jesperancinha.lyrics.test.Main.main;
import static org.junit.jupiter.api.Assertions.assertAll;

public class MainTest {

    @BeforeAll
    public static void beforeAll() {
        ClassLoader.getSystemClassLoader().setDefaultAssertionStatus(true);
    }

    @Test
    public void testMain() {
        assertAll(() -> main(null));
    }

}

File: favourite-lyrics-test/src/test/kotlin/org/jesperancinha/lyrics/test/MainTest.kt
package org.jesperancinha.lyrics.test

import org.jesperancinha.lyrics.test.Main.main
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.BeforeAll
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.function.Executable

class MainTest {
    @Test
    fun testMain() {
        Assertions.assertAll(Executable { main() })
    }

    companion object {
        @JvmStatic
        @BeforeAll
        fun beforeAll(): Unit {
            ClassLoader.getSystemClassLoader().setDefaultAssertionStatus(true)
        }
    }
}